src/dotnet-svcutil/lib/Microsoft.Svcutil.NamedPipeMetadataImporter/Microsoft.Svcutil.NamedPipeMetadataImporter.csproj

```diff
@@ -1,14 +1,40 @@
 <Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyVersionFile>$(IntermediateOutputPath)\$(TargetFramework)\$(MSBuildProjectName).$(TargetFramework).version.cs</AssemblyVersionFile>
+  </PropertyGroup>
 	<PropertyGroup>
-		<TargetFrameworks>net6.0;net462</TargetFrameworks>
+		<TargetFrameworks>net6.0;net8.0;net462</TargetFrameworks>
     <RootNamespace>Microsoft.Tools.ServiceModel.Svcutil</RootNamespace>
     <AssemblyName>$(MSBuildProjectName)</AssemblyName>
     <LangVersion>latest</LangVersion>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
 	</PropertyGroup>
 	<ItemGroup>
-		<PackageReference Include="System.ServiceModel.NetNamedPipe" Version="6.0.*" Condition="'$(TargetFramework)' == 'net6.0'">
-			<PrivateAssets>contentfiles;analyzers;build</PrivateAssets>
+		<PackageReference Include="System.ServiceModel.NetNamedPipe" Version="6.2.*" Condition="'$(TargetFramework)' == 'net6.0'">
+			<IncludeAssets>all</IncludeAssets>
 		</PackageReference>
```

Comment:
since this 8.0 version of dotnet-svcutil.exe, only need to reference 8.0 version of WCF Client library.

Reply:
If only reference the 8.0 NetNamedPipe package, it can cause issues for user project targeting net6.0, particularly when adding net.pipe service reference with type reuse option enabled. In such case, the associated bootstrapper project will also target net6.0, causing problem situation of loading the 8.0 NetNamedPipe package assembly in a net6.0 runtime environment.

We have two potential solutions::
1. Use the 6.2 named pipe package for net6.0 target and 8.0 named pipe package for the net8.0 target.
2. Use the 6.2 named pipe package for both net6.0 and net8.0 targets.

The current implementation adopts the 1st option.